docs/waivers.md

```diff
@@ -0,0 +1,54 @@
+# Waivers (Beta)
+
+Waivers is a mechanism to mark controls as "waived" for various reasons
+and to control the running and/or reporting of those controls. It
+takes a YAML input file that identifies which controls are waived,
+(optionally) whether they should be skipped from running, a description of why it
+is waived, and (optionally) an expiration date for the waiver.
+
+NOTE: This mechanism is currently in beta and is ultimately meant to be
+ingested and maintained by Chef Automate UI.
+
+## Usage
+
+To use waivers, you must have a correctly formatted input file and
+invoke `inspec exec` with `--waiver-file [path]`. Currentlty, waiver results are
+only reported with the JSON reporter type, so you must also have
```

Comment:
Waiver information is at least partially visible in the default CLI output as well.